app/report/report.service.js

```diff
@@ -0,0 +1,65 @@
+const _ = require('lodash');
+const moment = require("moment");
+const noticeService = require('../notice/notice.service');
+const Notice = require('../notice/notice');
+
+async function overview() {
+	return {
+		total: (await noticeService.get({}, true, true)).length,
+		requests: {
+			total: (await noticeService.get({ type: "request" }, true, true)).length,
+			active: (await noticeService.get({ type: "request" })).length
+		},
+		offers: {
+			total: (await noticeService.get({ type: "offer" }, true, true)).length,
+			active: (await noticeService.get({ type: "offer" })).length
+		}
+	}
+}
+
+async function periodic(start_date, end_date) {
+	return {
+		requests: {
+			total: (await Notice.find({
```

Comment:
You should avoid calling DB 4 times with similar requests in one function. Try to get all required data with single DB call.